src/Foundatio.Redis/Scripts/ReplaceIfEqual.lua

```diff
@@ -1,7 +1,7 @@
 ﻿local currentVal = redis.call('get', @key)
 if (currentVal == false or currentVal == @expected) then
   if (@expires ~= nil and @expires ~= '') then
-    return redis.call('set', @key, @value, 'PX', @expires) and 1 or 0
+    return redis.call('set', @key, @value, 'EX', @expires) and 1 or 0
```

Comment:
https://github.com/FoundatioFx/Foundatio.Redis/blob/4212c67514825db74625f0fa2eb4f08c670ad5fe/src/Foundatio.Redis/Cache/RedisCacheClient.cs#L397 Is passing it total seconds

Reply:
Looks good.